src/Appwrite/Utopia/Database/Validator/Queries/Base.php

```diff
@@ -20,9 +20,12 @@ class Base extends Queries
      * @param string $collection
      * @param string[] $allowedAttributes
      */
-    public function __construct(string $collection, array $allowedAttributes)
+    public function __construct(string $collection, array $allowedAttributes, array $collections = [])
```

Comment:
Let's add an explanation on the doc block, what is this param used for and what is the default behavior.

Reply:
done